SPECS/glibmm/glibmm.spec

```
@@ -0,0 +1,159 @@
+%global apiver 2.68
+
+Summary:        C++ interface to the glib
+Name:           glibmm
+%define BaseVersion 2.70
+Version:        %{BaseVersion}.0
```

I'd suggest keeping the full version in the `Version` tag and then extract other data you need from it. This way we have to modify only one place in the spec. Also, I'd keep macro definitions in one place, so they are easier to find.
```suggestion
%global apiver 2.68
%define BaseVersion %(echo %{version} | cut -d. -f1-2)

Summary:        C++ interface to the glib
Name:           glibmm
Version:        2.70.0
```